articles/openshift/oshift-how-build-app-private-repo.md

```diff
@@ -0,0 +1,119 @@
+---
+title: How to build an OpenShift application from a private Github repo | UKCloud Ltd
+description: Shows how to build an OpenShift application where the source code is in a "private" github repository secured with an SSH Key Pair.
+services: openshift
+author: Andy Watkins
+
+toc_rootlink: How To
+toc_sub1:
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Build an OpenShift app from a private Github repo
+toc_fullpath: How To/oshift-how-build-app-private-repo.md
+toc_mdlink: oshift-how-build-app-private-repo.md
+---
+
+# How to build an OpenShift application from a private Github repo
+
+This guide assumes you have access to the command-line `oc` client, and have
+logged in to your openshift instance with `oc login`
+
+With a private Github repo, you want to ensure it stays hidden from prying
+eyes, but you also want your Openshift app to be able to build from the repo.
+The way around this is to use a 
+[Deploy key](https://developer.github.com/v3/guides/managing-deploy-keys/#deploy-keys).
+
+
+### Steps:
+**First**
+create an ssh key-pair (don't use your regular one, and make sure
+you don't overwrite your regular one either!!)
```

Comment:
You could add this as a note on a new line. Syntax is covered here: https://docs.ukcloud.com/articles/other/other-ref-knowledge-guidelines.html#for-notes
Don't use exclamation points

Reply:
Thanks. I followed the instructions for Note syntax. I wasn't 100% sure if the leading greater-than sign was necessary until I read your last comment below. Can I suggest clarifying that in the guidelines doc?

Reply:
I'll update the guidelines to be clearer, thanks